tests/Earthfile

```diff
@@ -0,0 +1,27 @@
+VERSION 0.7
+
+FROM debian:stable-slim
+
+# BASH, CURL, GPG, NODEJS, NPM, JDK, JRE
+dependencies:
+    RUN apt-get update
+    RUN apt-get install bash
+    RUN apt-get install curl -y
+    RUN apt-get install gpg -y
+    RUN curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg
+    RUN apt-get install nodejs -y
+    RUN apt-get install npm -y
+    RUN apt-get install default-jre -y
+
+generate-client:
+    FROM +dependencies
+    
+    # TODO: GET RIGHT FILE FROM POEM GENERATION / BY COPY FILE IN REPO
```

Comment:
I would avoid having a todo in the code you are merging

Reply:
Why not ? If it is not aligned for the current ticket and could be a part of the future improvements

Reply:
I agree with @kukkok3 . TODOs are technical deps. We should avoid creating them.
If we fail to fully complete the task and must create a TODO, then a TODO should have an owner and GitHub ticket.

Reply:
I agree in general, but afaik the required process to generate the openapi spec via poem is not completed.
@stevenj do we have a ticket for poem implementation so that I can link this TODO to that ticket?